QualifiedTeachersApi/src/QualifiedTeachersApi/DataStore/Crm/DataverseAdapter.SetIttResultForTeacher.cs

```diff
@@ -329,8 +363,8 @@ public async Task<SetIttResultForTeacherLookupResult> LookupData()
             // in which case the status should be UnderAssessment, WithDrawn or Deferred.
 
             var inTrainingForProvider = ittRecords
```

Comment:
This variable name could with changing (since it's no longer only 'in training' records that are searched for).
Nit: lines are super long!